src/spikeinterface/core/recording_tools.py

```diff
@@ -302,7 +302,7 @@ def get_chunk_with_margin(
     return traces_chunk, left_margin, right_margin
 
 
-def order_channels_by_depth(recording, channel_ids=None, dimensions=("x", "y")):
+def order_channels_by_depth(recording, channel_ids=None, dimensions=("x", "y"), flip=False):
```

Comment:
Maybe using the standard reverse from the python API would be a good idea?

https://docs.python.org/3/library/functions.html#sorted

It is also supported by the list.sort

But maybe you think that flip describes it better?
Just throwing something that I thought.

Reply:
Ah, on the other hand we have:

https://numpy.org/doc/stable/reference/generated/numpy.flip.html

Which seems to be aligned withat what you are doing.

Reply:
Yep. I also think that your docstring is doing the heavy lifting there anyway.